docs/spec/openapi/apm-openapi.yaml

```diff
@@ -0,0 +1,162 @@
+openapi: 3.1.0
+info:
+  title: APM Server
+  description: OpenAPI schema for APM Server APIs
+  version: 8.13.2
+  license:
+    name: Elastic License 2.0
+    url: https://www.elastic.co/licensing/elastic-license
+  contact:
+    name: APM Team
+servers:
+  - url: /
+tags:
+  - name: agent config
+    description: APIs that query the APM Server for configuration changes.
+    x-displayName: APM agent configuration
+  - name: server info
+    description: APIs that query general APM Server information.
+    x-displayName: APM server information
+paths:
+  /:
+    get:
+      summary: Get general server information
+      description: |
+        This lightweight endpoint is useful as a server up/down health check.
+        To configure authenticated access to the APM server, the instructions at [APM API key](https://www.elastic.co/guide/en/observability/current/api-key.html) or [APM Secret Token](https://www.elastic.co/guide/en/observability/current/secret-token.html) must be followed to configure the correct permissions for APM access.
+      operationId: getServerHealth
+      tags:
+        - server info
+      responses:
+        '200':
+          $ref: '#/components/responses/200_server_info'
+    post:
+      summary: Get general server information
+      operationId: postServerHealth
+      tags:
+        - server info
+      responses:
+        '200':
+          $ref: '#/components/responses/200_server_info'
+  /config/v1/agents:
+    get:
+      summary: Get agent configuration changes
+      description: >
+        To configure authenticated access to the APM server, the instructions at [APM API key](https://www.elastic.co/guide/en/observability/current/api-key.html) or [APM Secret Token](https://www.elastic.co/guide/en/observability/current/secret-token.html) must be followed to configure the correct permissions for APM access.
+      operationId: getAgentConfig
+      tags:
+        - agent config
+      parameters:
+        - in: query
+          name: service.name
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: A successful response. 
+          content:
+            application/json:
+              schema:
+                type: object
+    post:
+      summary: Get agent configuration changes
+      description: >
+        To configure authenticated access to the APM server, the instructions at [APM API key](https://www.elastic.co/guide/en/observability/current/api-key.html) or [APM Secret Token](https://www.elastic.co/guide/en/observability/current/secret-token.html) must be followed to configure the correct permissions for APM access.
+      operationId: postAgentConfig
+      tags:
+        - agent config
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - service
+              properties:
+                CAPTURE_BODY:
```

Comment:
Q: what is `CAPTURE_BODY`?

Reply:
Good question!  I omitted descriptions from the OpenAPI document for now since I didn't know the answer to questions like that, but we'd definitely want to add descriptions to make it more useable than our existing docs.

Reply:
For now, I've added a description based on what appears in https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/apm/common/agent_configuration/setting_definitions/general_settings.ts